index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+    <meta charset="UTF-8">
+    <title>JavaScript Lesson1-4</title>
+</head>
+<body>
+    <script>
+      const tasks = {
+        '掃除' : '机を片付ける',
+        '買い物' : '牛乳を買う',
+        '運動' : '散歩する'
+      }
+
+      function showTasks() {
+        console.log('====================');
+        console.log('現在持っているタスク');
+        console.log('====================');
+
+        Object.keys(tasks).forEach((key, index) => {
+          console.log(`${index}:[内容]${tasks[key]}、[ジャンル]${key}`)
+        })
```

Comment:
`forEach` を使うほうが簡単だと思います。

```suggestion
    tasks.forEach((task, index) => {
      console.log(`${index}:[内容]${task.content}、[ジャンル]${task.genre}`)
    })
```